tracks/julia/exercises/difference-of-squares/mentoring.md

```diff
@@ -4,6 +4,24 @@
 
 ![](https://wikimedia.org/api/rest_v1/media/math/render/svg/5852a9aaf138078bfb5a60e1e6cad558b57a9c3e)
 
+- Analytical solutions for sum of the first N square numbers: [square pyramidal number](https://en.wikipedia.org/wiki/Square_pyramidal_number)
+
 ### Talking points
 
+- You might be interested to know that `sum(1:n)` already uses Gauss' formula. You can see this with `@edit sum(1:n)`
```

Comment:
```suggestion
- You might be interested to know that `sum(1:n)` already uses Gauss' formula. You can see this with `@less sum(1:n)`
```

This is personal preference but for only looking up definitions, `@less` might be nicer.

Reply:
I've found @edit less buggy 🤷‍♂️